packages/core/src/api/applicationCommands.ts

```diff
@@ -190,15 +241,18 @@ export class ApplicationCommandsAPI {
 	 * @see {@link https://discord.com/developers/docs/interactions/application-commands#bulk-overwrite-guild-application-commands}
 	 * @param applicationId - The application id to overwrite commands for
 	 * @param guildId - The guild id to overwrite commands for
-	 * @param data - The data to use when overwriting commands
+	 * @param body - The data to use when overwriting commands
+	 * @param options - The options to use when overwriting the commands
 	 */
 	public async bulkOverwriteGuildCommands(
 		applicationId: Snowflake,
 		guildId: Snowflake,
-		data: RESTPutAPIApplicationCommandsJSONBody,
+		body: RESTPutAPIApplicationCommandsJSONBody,
+		{ signal }: { signal?: AbortSignal | undefined },
```

Comment:
This needs to default, otherwise `bulkOverwriteGuildCommands(APP_ID, GUILD_ID, body)` will throw an error as it cannot deconstruct `signal` from `undefined`.

Also, without the `= {}`, this argument is required.